app/src/main/java/org/oppia/app/topic/play/TopicPlayFragment.kt

```diff
@@ -8,17 +8,39 @@ import android.view.ViewGroup
 import org.oppia.app.fragment.InjectableFragment
 import javax.inject.Inject
 
+internal const val KEY_CURRENT_EXPANDED_LIST_INDEX = "CURRENT_EXPANDED_LIST_INDEX"
+internal const val NO_INDEX = -1
```

Comment:
Let's use nullability to represent this (e.g. pass a null index if absent) rather than using a sentinel value. Using a null value for absence is a bit more Kotlin idiomatic.

Reply:
Updated here and everywhere this value was getting used.